controls/radchartview/features/annotations/labels.md

```diff
@@ -10,21 +10,232 @@ position: 8
 
 # Annotation Labels
 
-The CartesianStrokedAnnotation class provides infrastructure for annotation labels. Currently the __CartesianGridLine__ and __CartesianPlotBand__ annotations inherit it and provide out-of-the-box support for this feature.      
+The CartesianStrokedAnnotation class provides infrastructure for annotation labels.
 
-## Properties
+>  Only the [CartesianGridLine]({%slug radchartview-annotations-gridline%}) and [CartesianPlotBand]({%slug radchartview-annotations-plotband%}) annotations have out-of-the-box support for this feature.

```

Comment:
I think that it is useful to have the information that the CartesianPlotBand and CartesianGridLine inherit the CartesianStrokedAnnotation in this note.